JavaScript/form.js

```diff
@@ -0,0 +1,39 @@
+const form = document.querySelector('form');
+
+form.addEventListener('submit', function(event) {
+
+    const responses = [
+        document.getElementById('applicantName').value,
+        document.getElementById('applicantEmail').value,
+        document.getElementById('applicantAddress').value,
+        document.getElementById('applicantCity').value,
+        document.getElementById('applicantState').value,
+        document.getElementById('applicantZip').value,
+        document.getElementById('applicantFirst').value,
+        document.getElementById('applicantPickup').value
+    ];
+
+    let pass = true;
+
+    for (let i = 0; i < responses.length; i++) {
+        res = responses[i];
+        if (res == '') {
+            pass = false;
+        }
+        if (i === 1 && !responses[i].includes('@')) {
```

Comment:
This is a good start, but invalid addresses can still pass. ie : badEmail@badURL (no .com etc!)
There are many ways to validate, one being regex (Regular Expressions)
Here is one source I found on the subject: 
https://www.w3resource.com/javascript/form/email-validation.php
 But don't worry too much about it. regex is difficult even for experienced programmers and most people just google it!